roles/fio-distributed/templates/client.yaml

```diff
@@ -38,6 +38,14 @@ spec:
         command: ["/bin/sh", "-c"]
         args:
           - "cat /tmp/host/hosts;
+{% if fiod.rook_ceph.prefill is sameas true %}
```

Comment:
Modify conditional for something like this to avoid breaking when the variable is not set
```
{% if prefill is defined and prefill is sameas true %}
```

Reply:
ack. will make change